modules/network-observability-flowcollector-view.adoc

```diff
@@ -83,5 +86,6 @@ spec:
 ----
 <1> The Agent specification, `spec.agent.type`, must be `EBPF`. eBPF is the only {product-title} supported option.
 <2> You can set the Sampling specification, `spec.agent.ebpf.sampling`, to manage resources. Lower sampling values might consume a large amount of computational, memory and storage resources. You can mitigate this by specifying a sampling ratio value. A value of 100 means 1 flow every 100 is sampled. A value of 0 or 1 means all flows are captured. The lower the value, the increase in returned flows and the accuracy of derived metrics. By default, eBPF sampling is set to a value of 50, so 1 flow every 50 is sampled. Note that more sampled flows also means more storage needed. It is recommend to start with default values and refine empirically, to determine which setting your cluster can manage.
-<3> The Loki specification, `spec.loki`, specifies the Loki client. The default values match the Loki install paths mentioned in the Installing the Loki Operator section. If you used another installation method for Loki, specify the appropriate client information for your install.
-<4> The `spec.quickFilters` specification defines filters that show up in the web console. The `Application` filter keys,`src_namespace` and `dst_namespace`, are negated (`!`), so the `Application` filter shows all traffic that _does not_ originate from, or have a destination to, any `openshift-` or `netobserv` namespaces. For more information, see Configuring quick filters below.
\ No newline at end of file
+<3> The optional specifications `spec.processor.logTypes`, `spec.processor.conversationHeartbeatInterval`, and `spec.processor.conversationEndTimeout` can be set to enable conversation tracking. When enabled, conversation events are queryable in the web console. The values for `spec.processor.logTypes` are `FLOWS` to export `flowLog` or `CONVERSATIONS` to export `newConnection`, `heartbeat` and `endConnection` events alongside `flowLog`. `ENDED_CONVERSATIONS` to export only `endConnection` events or `ALL` to export both `flowLogs` and conversations events. Storage requirements increase by enabling this option.
```

Comment:
`CONVERSATIONS` dont have `flowLog` so that needs to be removed

Reply:
Honestly, I am thinking about removing all these UI values and leaving the UI explanation to the Observing network traffic topic, Working with conversation tracking